### DISCUSSION TITLE
Where do I use nanos for?

### DISCUSSION THREAD

**Discussion by WantToLearnJapanese** [nanovms/nanos#1739] [Q&A] [answered]
Where do I use nanos for?

Can I run nanos with docker as a container?

**Comment by francescolavra** [comment]
You can uses nanos wherever you want to run a cloud application and care about security and performance.
Nanos runs directly on a VM, without any lower-level software other than the hypervisor that manages the VM; as such, nanos cannot be run with docker, unikernels and containers are two different things (see https://nanovms.com/learn/docker-vs-unikernels).

**Reply by WantToLearnJapanese** [reply]
Oh, now I understand a little about the concept. 
By the way, can I beg you some concrete example usages of Nanos? 
I tried to google something like Docker Hub for Nanos but failed to find it.

**Comment by francescolavra** [comment, marked as answer]
The easiest way to use Nanos is via its orchestration tool [Ops](https://ops.city/). There are quite a few usage examples at https://github.com/nanovms/ops-examples.
The equivalent of Docker Hub for Nanos is the [Ops package hub](https://repo.ops.city/). If you look at the quick start example at https://ops.city/ (a node.js demo server), it makes use of a package from Pkghub.
